browser/js/common/factories/reviews-factory.js

```diff
@@ -0,0 +1,21 @@
+app.factory('ReviewsFactory', function($http) {
+	return {
+		fetchAllByUser: () => {
+			return $http.get('/api/users/:userId/reviews').then(reviews => {
```

Comment:
you need to update the route to this: "/api/users/"+userId+"/reviews"
and where is userId coming from?